<launch>
    <node name="move_and_back_odom" pkg="gen5" type="move_and_back_odom.py" output="screen"/>
    <node name="sigint_catcher" pkg="gen5" type="sigint_catcher.py" output="screen"/>
    <node name="rotate_server" pkg="gen5" type="rotate_server.py" output="screen"/>
    <node name="run_server" pkg="gen5" type="run_server.py" output="screen"/>
</launch>